Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM golang:_GOVERSION_
+
+RUN echo "deb http://deb.debian.org/debian stretch main" >> /etc/apt/sources.list
+
+# gcc for cgo
+RUN apt-get update && apt-get install --force-yes -y --no-install-recommends \
+		g++ \
+		gcc \
+		libc6-dev \
+		make \
+		pkg-config \
+		gcc-aarch64-linux-gnu \
+		libc6-dev-arm64-cross \
+		libblkid-dev \
+		sudo \
+		uuid-runtime \
+		gdisk \
+		kpartx \
+		e2fsprogs \
+		dosfstools \
+		file \
+	&& rm -rf /var/lib/apt/lists/*
+
+ENV GOLANG_VERSION _GOVERSION_
+
+RUN echo '%jenkins ALL=NOPASSWD:ALL' >> /etc/sudoers;
+
+ENV GOPATH /go
```

Comment:
The golang images should all be setup with the right environment variables I believe. Here's my builder (admittedly different packages are needed):

```
FROM golang:1.8.3-alpine3.6

RUN apk add --no-cache ca-certificates curl bash git make openssh \
  && rm -rf /var/cache/apk/*
```